md/profileCreation.md

```diff
@@ -20,23 +32,22 @@ A **file of profiles** is an XML file containing one or more profiles.
 	</profile>
 </profiles>
 ```
-
 However, profiles in a given file of profiles are totally **independent**.  
-The purpose of grouping profiles in one file is to manage together profiles related to a same business application.  
-For example, if you want to create a leave management application for four kind of users (Employees, Managers, Human Resources officers, and Administrators) then you will have to create four applications and four profiles, and to bind each profile to the related application. As for the four applications, the four profiles should be managed together, and so grouped in the same file of profile. 
-Files of profiles are also handy to be used in different environments: you can create different files, with the same profiles, but each profile is mapped to different entities of the organization depending on the target environment.
-Therefore, you could have a "profiles_dev.xml", a "profiles_UAT.xml", and a "profiles_prof.xml" files of profiles.
 
-There is a special file of profiles for default profiles: `default_profiles.xml`  
-This file can only contain default profiles and cannot be renamed or deleted.
-Only the mapping between default profiles and entities of the organization can be edited.
+Default profiles belong to the file `default_profiles.xml`. 
```

Comment:
I think it would be nice to specify: "Default profiles (administrator, user...)"

Reply:
ok